test/libsolidity/semanticTests/structs/struct_copy.sol

```diff
@@ -40,7 +40,7 @@ contract c {
 // set(uint256): 7 -> true
 // gas irOptimized: 101963
 // gas legacy: 102216
-// gas legacyOptimized: 101606
+// gas legacyOptimized: 101744
```

Comment:
Here is the diff: https://gist.github.com/hrkrshnn/7ab08d75e0f1a718a53c7d0f564bc9e6